warehouse/static/js/warehouse/utils/webauthn.js

```diff
@@ -159,16 +159,23 @@ const postAssertion = async (assertion, token) => {
   return await resp.json();
 };
 
-export const GuardWebAuthn = () => {
-  if (!window.PublicKeyCredential) {
+export const GuardWebAuthn = async () => {
+  if (
+      !window.PublicKeyCredential ||
+      !(
+          await window
+              .PublicKeyCredential
+              .isUserVerifyingPlatformAuthenticatorAvailable()
```

Comment:
This is not correct -- `isUserVerifyingPlatformAuthenticatorAvailable` checks for an authenticator that's part of the hardware (e.g. touchid), but webauthn works fine without those.

Reply:
Right. I already see that it breaks on my Google Chrome for Android of version 75. Have to invent something more creative then.